src/olympia/editors/views.py

```diff
@@ -625,14 +654,24 @@ def version(self):
 @addons_reviewer_required
 @addon_view_factory(qs=Addon.unfiltered.all)
 def review(request, addon, channel=None):
+    content_review_only = False
     # channel is passed in as text, but we want the constant.
+    if channel == 'content':
+        # 'content' is not a real channel, just a different review mode for
+        # listed add-ons.
+        content_review_only = True
+        channel = 'listed'
     channel = amo.CHANNEL_CHOICES_LOOKUP.get(
         channel, amo.RELEASE_CHANNEL_LISTED)
     unlisted_only = (channel == amo.RELEASE_CHANNEL_UNLISTED or
                      not addon.has_listed_versions())
     if unlisted_only and not acl.check_unlisted_addons_reviewer(request):
         raise PermissionDenied
 
+    if content_review_only and not acl.action_allowed(
```

Comment:
nit: we can fail earlier, before we have to do a db query for `addon.has_listed_versions()`